src/onnx/onnx_parser.cpp

```diff
@@ -283,9 +290,11 @@ void onnx_parser::parse_graph(module* mod, const onnx::GraphProto& graph)
         }
         else
         {
-            result = ops[node.op_type()](
-                *this, {get_attributes(node), output_num, node.op_type(), mod}, args);
+            std::string node_name = node.op_type() + "_" + std::to_string(node_index);
+            result                = ops[node.op_type()](
+                *this, {get_attributes(node), output_num, node_name, mod}, args);
         }
+        ++node_index;
```

Comment:
Can `instructions.size()` be used for this instead?

Reply:
fixed